app/stac_api/views.py

```diff
@@ -349,6 +382,14 @@ def get_success_headers(self, data):
         asset_link_self = self.request.build_absolute_uri() + "/" + self.request.data["id"]
         return {'Location': asset_link_self}
 
+    def get_serializer(self, *args, **kwargs):
```

Comment:
As this function is being uses several times, it could be defined once outside the class?

Reply:
Makes sense, I'll see if it works to put it in the `mixins_views.py`

Reply:
Done. Introduced class `BaseAPIView` in `views.py` with overriden `get_serializer()`.